models/stg_marketo__lead.sql

```diff
@@ -12,22 +12,16 @@ with base as (
                 staging_columns=get_lead_columns()
             )
         }}
+        -- This will check if there are non-default columns to bring in
+        {% set default_cols = ['id', 'created_at', 'updated_at', 'email', 'first_name', 'last_name'] %}
+        {% set new_cols = dbt_utils.star(from=ref('stg_marketo__lead_tmp'), except=default_cols) %}
+        {% if new_cols != '/* no columns returned from star() macro */' %}
```

Comment:
This is really neat! We definitely will want to keep an eye on this macro in the dependent package to ensure this string doesn't change in the future.